Polyfills/Window/Source/Window.cpp

```diff
@@ -88,10 +135,8 @@ namespace
         {
             {
                 std::unique_lock<std::mutex> lk(m_mutex);
-                while (!m_queue.empty())
-                {
-                    m_queue.pop();
-                }
+                m_idMap.clear();
+                m_timeMap.clear();
                 m_shutdown = true;
                 m_condVariable.notify_one();
```

Comment:
Probably need to call `m_thread.join()` to make sure the thread is done using member variables before the destructor completes.

Reply:
Thread is joined further down in the destructor:

https://github.com/docEdub/BabylonNative/blob/3a67ac6606074ac01554aed372747b215041a565/Polyfills/Window/Source/Window.cpp#L144-L147

Reply:
Ahh, yes, GitHub hid it because I'm only looking at the commits you made.